common/docsV1.md

```diff
@@ -2116,3 +2130,78 @@ Learn more about the HL7 FHIR Specification for:
+
+**DPC v2:** The sandbox experience has been redesigned to allow Health IT implementers the capability of generating client tokens and uploading public keys in the sandbox environment without burdening providers with technical requests before a DPC solution has been approved. The DPC v2 sandbox portal also allows implementers the ability to manage credentials as a team on behalf of multiple healthcare organizations under one account and simplifies the process of downloading sample data from DPC's website.
+
+This workflow enables Health IT implementers to collaborate as a team and establishes best practices between healthcare organizations and their approved Health IT teams.
+
+## Attribution:
+After speaking with DPC’s sandbox users and production pilot partners, research insights indicated that the DPC v1 process for maintaining patient lists on behalf of providers proved more complex and stringent than other existing standards in the healthcare industry today.
+
+**Patient Lists**
+
+**DPC v1:** Users were asked to create and maintain Patient and Practitioner resources with each Practitioner requiring a Group of their own. Patient records were also required to be refreshed every 90 days.
+
+**DPC v2:** There are fewer Group-related API endpoints as implementers no longer need to maintain the age of FHIR resources in v2. Instead, implementers are required to generate a new list of patients using the <a href="/Create a Group" target="_blank"> Create a Group</a> endpoint (link) that will reference Patient MBIs (Medicare Beneficiary Identifier) in conjunction with their practitioner's Provider NPI (National Provider Identifier) in every export request. Groups may contain patients for multiple practitioners in one request. This method of generating a new list every time 
+
+For every subsequent data export, a new Group is needed. This approach simplifies the user experience, limits the number of endpoints being used, ensures that the data being requested on behalf of healthcare organizations is the most updated list of active beneficiaries receiving treatment at the time of request, and creates a path for long-term auditing purposes.
+
+**Patient identifiers**
+
+**DPC v1:** Requested the UUID of a patient provided by the DPC API.
+
+**DPC v2:** Requires the beneficiary’s MBI. 
+
+## Attestation/Provenance Header:
+
+**DPC v1:** Both the organization and the practitioner were to be included in the provenance header. 
+
+**DPC v2:** Only the organization ID is required in the provenance header (link to v2 Provenance section), since a Group will contain multiple patients and practitioners on behalf of a single organization in one request. 
```

Comment:
We should check that this link works, but we need one 😉 
```suggestion
**DPC v2:** Only the organization ID is required in the <a href="/Attestation%20%26%20Attribution" target="_blank">provenance header</a>, since a Group will contain multiple patients and practitioners on behalf of a single organization in one request. 
```